src/Hero.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Random;
+
+public class Hero {
+    Random generator = new Random();
+    public Coordinates position = new Coordinates(0, 0);
+    char symbol;
+
+    public Hero(int height, char[][] matrix) {
+        setInitialPosition(height, matrix);
+        symbol = 'H';
+    }
+
+    private void setInitialPosition(int height, char[][] matrix) {
```

Comment:
Mega ciekawe że pomyślałeś o tym żeby Hero miał mapę, to jest trochę na odwrót niż zrobiłaby większość osób (mapa ma bohatera). Ale bardzo dobry instykt, nie wiem czy już słyszałeś o SOLID, ale jak nie to jestem pod mega wrażeniem.
Nie jest to oczywiście jedyny sposób żeby to zrobić, ale broni się.

Reply:
Ciekawe - nie słyszałem o SOLID. Czy dobrze rozumiem, że gdyby "mapa miała bohatera" to byłoby coś w rodzaju map.setInitialPostion(hero) ? Nawet o tym nie myślałem ;)